packages/server-admin-ui/src/views/ServerConfig/Settings.js

```diff
@@ -234,6 +234,24 @@ class Settings extends Component {
                       files in Multiplexed format in this directory
                     </FormText>
                   </Col>
+              </FormGroup>
+              <FormGroup row>
+                  <Col md='2'>
+                    <Label htmlFor='pruneContextsMinutes'>
+                      Remove Lost AIS Targets
```

Comment:
I would reword this so that AIS is mentioned but not the primary thing - other contexts can come from non-AIS sources, like your cloud thing. Must include unit.

_Maximum age of inactive vessels' data in minutes_ maybe? Reword the other text to match?